enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/CypherComparisonSupport.scala

```diff
@@ -524,12 +524,14 @@ object CypherComparisonSupport {
 
```

Comment:
Nice work here with all the configurations! Looks correct as far as I can see :)